example/FFTConvolutionOverRegionOfInterest.ipynb

```diff
@@ -0,0 +1,665 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Generate an input image and visualize ROI\n",
+    "\n",
+    "\n",
+    "def make_image() -> image_type:\n",
+    "    arr = np.ones([150, 150]).astype(np.float32)\n",
```

Comment:
`dtype` can be specified as a parameter to `np.ones`.

Reply:
Amended